spec/timezone.html

```diff
@@ -327,6 +327,14 @@ <h1>FormatTimeZoneOffsetString ( _offsetNanoseconds_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-timezonefrom" aoid="TimeZoneFrom">
+      <h1>TimeZoneFrom ( _identifier_ )</h1>
+      <emu-alg>
+        1. Let _from_ be ? Get(%Temporal.TimeZone%, *"from"*).
```

Comment:
Can this be %Temporal.TimeZone%.from or does it have to be %Temporal%.TimeZone.from for full monkeypatching purposes?

Reply:
Wait, we allow monkey-patching `Temporal.TimeZone`?

Reply:
I didn't mean to suggest that we allow it or disallow it, but you certainly _can_ replace the `Temporal.TimeZone` constructor; it's a question of whether we want this to get the `from` property from the original or the replaced one.

Reply:
I think allowing monkey-patching for top level classes is a can of worms we don't really want to open.

Reply:
This definitely has to be `%Temporal.TimeZone.from%` and not do an observable lookup.

Reply:
No, monkeypatching `from` is how SES locks down the time zone database, so it does need to do an observable lookup in any case.

Reply:
oof, that's something we'll want to discuss in committee. I thought the purpose of the `now` namespace was to make that be the only place SES had to monkeypatch, and that there were no places with an observable lookup.

Reply:
OK. Previous discussion is [here](https://github.com/tc39/proposal-temporal/blob/main/meetings/agenda-minutes-2020-05-07.md#294-idtocalendar--idtotimezone-follow-up-after-discussing-with-ses).